web/views.py

```
@@ -7168,6 +7186,451 @@ def delete_post(request, post_id):
+    }
+
+    return render(request, "membership/checkout.html", context)
+
+
+def change_membership_plan(_request: HttpRequest) -> HttpResponse:
+    """Redirect to membership plans page for changing plans."""
+    return redirect("membership_plans")
+
+
+@csrf_exempt
+def membership_webhook(request) -> HttpResponse:
```

Ruff (0.8.2)

Missing type annotation for function argument `request`

(ANN001)